dtran/dcat/api.py

```diff
@@ -205,10 +218,10 @@ def register_dataset_with_multiple_resources(
         return dataset, resources, variables
 
     def register_special_variables(
-        self, variable_data: List
+        self, variable_data: dict
     ):
         if not variable_data:
-            return []
```

Comment:
this PR doesn't seem to touch the DCat API, why we have those changes?

Reply:
Fixed this. Must have been a typo